cmd/apmbench/main.go

```diff
@@ -0,0 +1,34 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package main
+
+import (
+	"flag"
+	"log"
+	"testing"
+)
+
+// init the testing flags
+var _ = func() bool {
+	testing.Init()
+	return true
+}()
```

Comment:
```suggestion
// init the testing flags
func init() {
	testing.Init()
}
```

Reply:
Thanks, fixed now. I was trying to get the testing flags to work directly by initializing them before `init` but that didn't work as expected and I forgot to revert 🤦‍♂️